src/transport/raw/tests/TestTCP.cpp

```diff
@@ -56,19 +57,29 @@ TestContext sContext;
 const char PAYLOAD[]        = "Hello!";
 int ReceiveHandlerCallCount = 0;
 
-void MessageReceiveHandler(const PacketHeader & header, const Transport::PeerAddress & source, System::PacketBufferHandle msgBuf,
-                           nlTestSuite * inSuite)
+class MockTransportMgrDelegate : public TransportMgrDelegate
```

Comment:
Would it be worth it to now move this to some common header (a test delegate designed to check a specific source/destination/message id)?

Reply:
If move this one, several global variables need to be encapsulated, I'd like to do it in another PR.